app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -20,10 +20,11 @@ def connect(options = {})
     host       = options[:host] || address
     port       = options[:port] || self.port
     auth_type  = AUTH_TYPES[options[:auth_type]]
+    user_agent_label = "ManageIQ/#{MiqServer.first.version}"
```

Comment:
@rodneyhbrown7 @ManasaRaoK using cloudforms `MiqServer.first.version` will show the version of the cloudforms? Doesn't make sense to me expose the branch in this label. Do you have any particular request for this?

Reply:
It doesn't make any sense for me...

Reply:
The goal is to be able to identify what client was communicating with LXCA.    The version is just a way to identify what version of the client was used.

Reply:
Oooh, now it's clear for me! Thanks @rodneyhbrown7 :D

Reply:
So, @AndreyMenezes, @walteraa and @rodneyhbrown7 Could you review this PR, please? :+1:

Reply:
@rodneyhbrown7 I understand this, but for ManageIQ this call will get `fine-3`, `fine-4`, etc, depends on the branch of manageIQ. In cloudforms this call will return 4.5, 4.6, etc? Just to understand the expectancy.

Reply:
We should change the user_agent_label to "CFME/#{MiqServer.first.version}"